hal/armv8m/nrf/91/_init.S

```diff
@@ -0,0 +1,96 @@
+/*
+ * Phoenix-RTOS
+ *
+ * plo - operating system loader
+ *
+ * Low-level initialization for Cortex-M33 (ARMv8) architecture
+ *
+ * Copyright 2012, 2016-2017, 2020-2022 Phoenix Systems
+ * Author: Jacek Popko, Pawel Pisarczyk, Jakub Sejdak, Aleksander Kaminski, Damian Loewnau
+ *
+ * This file is part of Phoenix-RTOS.
+ *
+ * %LICENSE%
+ */
+
+#define __ASSEMBLY__
+
+#include "config.h"
+
+.syntax unified
+
+.section .init, "x"
+
+.globl _init_vectors
+.type _init_vectors, %object
+_init_vectors:
+.word _stack
+.word _start
+
+.word _exceptions_dispatch  /* NMI */
+.word _exceptions_dispatch  /* HardFault */
+.word _exceptions_dispatch  /* MemMgtFault */
+.word _exceptions_dispatch  /* BusFault */
+.word _exceptions_dispatch  /* UsageFault */
+.word _exceptions_dispatch  /* SecureFault */
+.word 0
+.word 0
+.word 0
+.word 0                     /* SVC */
+.word 0                     /* Debug */
+.word 0
+.word _interrupts_dispatch  /* PendSV */
+.word _interrupts_dispatch  /* Systick */
+
+.rept 217
```

Comment:
How many vectors does this MCU have? I think this is copied from STM32, most likely this one has less

Reply:
It seems that last vector corresponds to the last nrf9160 peripheral (ID = 66), so I will set 67 here.

![Screenshot from 2022-11-04 18-50-17](https://user-images.githubusercontent.com/77304431/200044317-9c1ec312-5d69-422d-b67f-b06a913fcb09.png)

![Screenshot from 2022-11-04 18-51-25](https://user-images.githubusercontent.com/77304431/200044326-79747cb8-af75-4158-8cb2-97dfbe4cd630.png)